packages/app/utils/authGuard.tsx

```diff
@@ -4,21 +4,39 @@ import { userAtom } from "app/state/user";
 import { useRouter } from "solito/router";
 import { SplashScreen } from "app/features/splash/splashScreen";
 
-export function AuthGuard({ children }: { children: ReactNode }) {
+type Props = {
+  children: ReactNode;
+  redirect?: boolean;
+  fallback?: ReactNode;
```

Comment:
```suggestion
  fallback?: () => ReactNode;
```

Reply:
If we do it like this we will lose the ability to use fallback with props

Reply:
I guess we don't need that option right now anyway.